README.md

```diff
@@ -5,11 +5,94 @@
+* `ADMIN_ROLE_SLUG`: Slug of the admin role (will be created on first run - default: admin)
+* `ADMIN_SCOPE_NAME`: Name of the admin scope (will be created on first run - default: admin)
+
+#### Additional options
+
+Additional options you might want to configure
+
+##### OIDC
+
+For OIDC, a customized [mozilla-django-oidc](https://github.com/mozilla/mozilla-django-oidc) is used.
+
+Most of the settings below are documented in it's [respective documentation](https://mozilla-django-oidc.readthedocs.io/en/stable/settings.html).
+
+* `OIDC_OP_USER_ENDPOINT`: Url of userinfo endpoint as [described](https://openid.net/specs/openid-connect-core-1_0.html#UserInfo)
+* `OIDC_VERIFY_SSL`: Bool (default: true)
+* `OIDC_USERNAME_CLAIM`: Name of claim that contains the username (default: sub)
+* `OIDC_EMAIL_CLAIM`: Name of claim that contains the email address (default: email)
+* `OIDC_BEARER_TOKEN_REVALIDATION_TIME`: Time in seconds before bearer token validity is verified again. For best security, token is validated on each request per default. It might be helpful though in case of slow Open ID Connect provider to cache it. It uses [cache](#cache) mechanism for memorizing userinfo result. Number has to be lower than access token expiration time. (default: 0)
+* `OIDC_CREATE_USER`: Enables or disables automatic user creation during authentication (default: false)
+* `OIDC_OP_INTROSPECT_ENDPOINT`: Url of introspection endpoint (optionally needed for Client Credentials Grant)
+* `OIDC_RP_CLIENT_ID`: ID of the client (optionally needed for Client Credentials Grant)
+* `OIDC_RP_CLIENT_SECRET`: Secret of the client (optionally needed for Client Credentials Grant)
+
+##### Cache
+
+* `CACHE_BACKEND`: [cache backend](https://docs.djangoproject.com/en/1.11/ref/settings/#backend) to use (default: django.core.cache.backends.locmem.LocMemCache)
+* `CACHE_LOCATION`: [location](https://docs.djangoproject.com/en/1.11/ref/settings/#std:setting-CACHES-LOCATION) of cache to use
+
+##### CORS headers
+
+Per default no CORS headers are set but can be configured with following options.
+
+* `CORS_ORIGIN_ALLOW_ALL`: If True, the whitelist will not be used and all origins will be accepted. (default: False)
+* `CORS_ORIGIN_WHITELIST`: A list of origin hostnames (including the scheme and with optional port) that are authorized to make cross-site HTTP requests.
+
+##### Logging
+
+* `LOG_LEVEL`: The logging level of the console logging handler (default: INFO)
+
+##### Anonymous write
+
+* `ALLOW_ANONYMOUS_WRITE`: If set to `true`, unauthenticated users are allowed to write data (default: false)
+
+Their request will also go through the permission layer and access can be restricted/denied from there.
+
+This setting can be handy when setting up emeis or during dev, when no OIDC provider is available.
+
+There might be a few use cases where this has to be used in production. For most use cases this
+should never be set to `true` in production environments.
+
```

Comment:
If we're not mentioning the cases where this is needed in production, we might as well write 

```suggestion
Should always be `false` in production environments.
```

Reply:
Done.